crawler/src/main/resources/application.conf

```diff
@@ -0,0 +1,27 @@
+akka {
+  quartz {
+    defualtTimezone = "Europe/Warsaw"
+
+    schedules {
+      EverydayMidnight {
+        description = "Everyday at midnight"
+        expression = "0 0 0 * * ?"
+      }
+    }
+  }
+}
+
+bootstrap {
+  servers = "localhost:9092"
+}
+
+crawler {
+  http {
+    timeout = 10s
+    max.parallelism = 30
+  }
+}
+
+kafka {
+  topic = "newspaper-content"
+}
```

Comment:
How about adding `include "environment.conf"` to make testing easier for Patryk? :)